tests/gfoidl.Base64.Tests/Assembly.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Text.RegularExpressions;
+using NUnit.Framework;
+
+namespace gfoidl.Base64.Tests
+{
+    [TestFixture]
+    public class Assembly
```

Comment:
The easiest way, but strictly speaking this shouldn't be a unit-test.
I know `sn.exe -T` shows the publickey-token, but this would need some surrounding code to verify the values, but this would be a nice solution.

Reply:
Ah, KISS. Leave it as is.